gcloud/datastore/test_client.py

```diff
@@ -620,8 +620,7 @@ def test_put_multi_no_batch_w_partial_key(self):
         client = self._makeOne(credentials=creds)
         client.connection._commit.append(_CommitResult(key))
 
-        result = client.put_multi([entity])
-        self.assertTrue(result is None)
+        client.put_multi([entity])
```

Comment:
The error was a weird one. I re-enabled this specific one to get the error message:

```
************* Module gcloud.datastore.test_client
E:623, 8: Assigning to function call which doesn't return (assignment-from-no-return)
```

We can disable it for our tests config? I just wanted `pylint` to be quiet.

I didn't really see the utility in checking the `None`.